src/mango_idx_view.erl

```diff
@@ -112,7 +112,12 @@ is_usable(Idx, Selector) ->
     % a member of the indexable fields of the selector.
     Columns = columns(Idx),
     Fields = indexable_fields(Selector),
-    lists:member(hd(Columns), Fields).
+    lists:member(hd(Columns), Fields) and not is_text_search(Selector).
```

Comment:
I'm not a big fan of how this works. I'd rather see is_text_search be its own thing that searches through the normalized selector for the $default field directly rather than trying to abuse the indexable_fields functions.